pages/common/virt-install.md

```diff
@@ -0,0 +1,8 @@
+# virt-install
+
+> libvirt utility to create new virtual machine and run installation.
+> Home page: <https://virt-manager.org/>.
```

Comment:
I think the linting error is here. I can certainly see why a lowercase letter would be preferred here though. What about this instead?
```suggestion
> Create new virtual machines with libvirt, installing the operating system automatically.
> More information: <https://virt-manager.org/>.
```

Reply:
Well, that is a good fix for linter. But I have a couple of comments:
- Utility does not actually _intall_ OS, it only boots an image, that is the point.
- Home page actually does not provide a lot of information, while [redhat.com page](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/virtualization_host_configuration_and_guest_installation_guide/sect-virtualization_host_configuration_and_guest_installation_guide-guest_installation-creating_guests_with_virt_install) does, but I am not sure it is an appropriate link.